src/css/tabs-dark/receiver_msp-dark.css

```diff
@@ -0,0 +1,7 @@
+.control-gimbal {
+  background-color: var(--subtleAccent);
+}
+
+.control-gimbal .crosshair {
+  background-color: var(--mutedText);
```

Comment:
I think it is better to not add an additional `-dark.css` file. You could put these variables into the `receiver_msp.css` file. If necessary add new variables.

Reply:
Here I changed the colors, therefore I add all these stuff into separate (-dark) files. Which are disabled/enabled when changing theme.
To be honest, this is a very strange decision to disable/enable a bunch of files. I would rather give or take a "dark_theme" class to <body> and then use ".dark_theme .whatever {..}" in css..

Reply:
I guess the point back then was to have everything separated to be able to easily remove dark mode if it gets outdated. But now since a lot of users actually use it and it is properly maintained I think it is best to slowly move from using -dark.css files to using variables since the only thing changing is the background, border, text color. That's why I said it is best to not add this separate file if it isn't necessary like in this case.

Reply:
Could you please briefly describe how you want to switch between two themes without -dark files? I can't catch it.

Reply:
Use the variables for classes where colors change in the non -dark.css file. Just like it is done with almost all borders and some other places like gui_box titlebars. Variable values get changed when main-dark.css is applied.

Reply:
Done. Had to introduce 2 variables for the gimbals. Accidentally noUi-slider was styled across the whole app :)